Source/Terminals/Forms/OrganizeFavoritesForm.cs

```diff
@@ -260,9 +260,9 @@ private void DeleteSelectedFavorites()
 
         internal static bool AskIfRealyDelete(string target)
         {
-            string messsage = string.Format("Do your realy want to delete selected {0}?", target);
+            string messsage = string.Format("Do you really want to delete selected {0}?", target);
             return MessageBox.Show(messsage, "Terminals - Delete",
-                MessageBoxButtons.YesNo, MessageBoxIcon.Question) == DialogResult.Yes;
+                MessageBoxButtons.YesNoCancel, MessageBoxIcon.Question) == DialogResult.Yes;

```

Comment:
Why YesNoCancel, if Cancel makes no sence here?

Reply:
It is sensefull fix. Because of user can push fast key escape on keyboard to cancel. It is much more usefull way than push dialog button with mouse